corehq/apps/data_cleaning/models.py

```diff
@@ -257,3 +259,28 @@ class BulkEditChange(models.Model):
 
     class Meta:
         ordering = ["created_on"]
+
+    def edited_value(self, case):
+        simple_transformations = {
+            EditActionType.REPLACE: lambda x: self.replace_string,
+            EditActionType.FIND_REPLACE: lambda x: x.replace(self.find_string, self.replace_string),
```

Comment:
does this support regular expressions?
https://github.com/dimagi/commcare-hq/blob/016784960037d260bcc7ca66eb3ed06f42bd4b24/corehq/apps/prototype/forms/data_cleaning.py#L334-L338

Reply:
No. Didn't we agree regexes were out of scope for v1? https://docs.google.com/document/d/1mboGuJDQmC3zREoGtWmUHRrZyDX7S1JlEZ7HSuLowfc/edit?disco=AAABawlbueE

Reply:
`self.use_regex` will note whether the `self.find_string` is a regular expression

Reply:
@orangejenny regexes were out of scope for filters, but not find/replace

Reply:
I'll add it in a future PR. The purpose of this PR is to get enough functionality that backend tests can run.